apps/webcert/src/components/header/WebcertHeader.tsx

```diff
@@ -71,6 +80,17 @@ const WebcertHeader: React.FC<Props> = ({ isEmpty = false }) => {
       'Abonnemang för Webcert saknas. Du har endast tillgång till Webcert för att läsa, skriva ut och makulera eventuella tidigare utfärdade intyg.',
     priority: 'MEDEL',
   }
+  const warningNormalOrigin: Banner = {
```

Comment:
Should this be placed here or does it belong in some utils class instead?

Reply:
I decided to follow the same pattern as for MissingSubscriptionBanner